prow/cmd/branchprotector/request.go

```diff
@@ -58,7 +58,7 @@ func makeBool(val *bool) bool {
 // makeChecks renders a ContextPolicy into the corresponding GitHub api object.
 //
 // Returns nil when input policy is nil.
-// Otherwise returns non-nil Contexts (empty if unset) and Strict iff Strict is true
```

Comment:
nit: `iff` is a [thing](https://en.wikipedia.org/wiki/If_and_only_if), looks like this property did not change?